### ISSUE TITLE
0.10.2 identifies itself as 0.10.1, then requests to be updated to 0.10.2 (!)

### ISSUE THREAD

**Issue by thegreyshadow** [flameshot-org/flameshot#2119]
### Flameshot Version

Flameshot Version 0.10.2

### Installation Type

Compiled from source

### Operating System type and version

Slackware, 64-bit, 15.0 RC2 (Slackware-current)

### Description

I downloaded and built (without problems) the 0.10.2 source tarball.
However, upon running flameshot I get an update prompt. Upon inspection of the systray icon, I get a notice: "New version 0.10.2 is available" and upon selecting the "About" menu entry, I am told that I'm running flameshot version 0.10.1.

### Steps to reproduce

1. Download the 0.10.2 source tarball
2. build it
3. install it
4. run it
and you will get the prompt

### Screenshots or screen recordings

![2021-11-25_17-34](https://user-images.githubusercontent.com/8303713/143497713-1d283e7d-9791-4e67-93d9-3794942f14bf.png)

Screenshot where it is seen that my source tarball is 0.10.2 while the app itself identifies as 0.10.1

### System Information

1, OS: Slackware Linux 64-bit 15.0 RC2 (Slackware-current, development version)
2. Monitor: Laptop screen, 1920x1080, 60 Hz
3. Desktop environment: Plasma (5.23.3 using Plasma Frameworks 5.88.0). Window manager: kwin (5.23.3) using Wayland.

**Comment by mmahmoudian** [comment]
What is the sha256sum of the source code you downloaded? Can you provide the link?

I believe you mistakenly think that if you download the source code from GitHub master branch you have downloaded v0.10.2 . The v0.10.2 was a hot fix for wayland users and is not in-line with what has been committed to master. It is more or less a cherrypick of certain commits.

So in order for us to check your bug report, we would need the sha256sum of the source you downloaded and the link from which you download the source code.

**Comment by farseerfc** [comment]
same here, the version number is not bumped for commit 2bf8a92 , which is tagged as v0.10.2 , and the appdata is also not updated. the [branch v0.10.2](https://github.com/flameshot-org/flameshot/tree/v0.10.2) appearently has a commit c4081c65c5d3d7c7579d07969c9f591077035d65 that is not included in the tagged release.

**Comment by borgmanJeremy** [comment]
Thanks for reporting. I'm going to pin this issue because I think it affects a few different packages.

I really really do not like having a maintenance release, but we needed to rush a fix out the door for gnome41.  Our build pipeline are all geared towards releasing off the main branch and I clearly didn't patch them properly on the maintenance branch.

I know this is annoying but you can disable the popup. I'm not going to spend any time trying to fix this since a.) it's soul crushing to work on backports b.) it will be resolved when v11 is ready (TBD when that is).

tldr: I apologize that this is really annoying but I'm simply not motivated to spend time fixing it.

**Comment by farseerfc** [comment]
Ok, understand. Switching to use the branch for the packaging for this release.

**Comment by thegreyshadow** [comment]
> 

sha256sum of my downloaded source tarball:

```951c8170b7be2f12e4446f6367e8fa176eb3489fe24c0cef8dfda0bd180e61f9  ./flameshot-0.10.2.tar.gz```

The link from which I downloaded:

https://github.com/flameshot-org/flameshot/archive/refs/tags/v0.10.2.tar.gz

**Comment by xvitaly** [comment]
I can confirm this issue on Fedora:

```
Flameshot v0.10.1
Compiled with Qt 5.15.2
```

Built from tag v0.10.2.

**Comment by xvitaly** [comment]
GNU/Linux downstreams can partially backport c4081c65c5d3d7c7579d07969c9f591077035d65 commit:

```diff
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.13)
 # cmake_policy(SET CMP0076 OLD)
 
-set(FLAMESHOT_VERSION 0.10.1)
+set(FLAMESHOT_VERSION 0.10.2)
 
 # Flameshot-org
 set(GIT_API_URL "https://api.github.com/repos/flameshot-org/flameshot/releases/latest")
--- a/data/appdata/org.flameshot.Flameshot.metainfo.xml
+++ b/data/appdata/org.flameshot.Flameshot.metainfo.xml
@@ -9,6 +9,7 @@ SPDX-License-Identifier: CC0-1.0
   <project_license>GPL-3.0-or-later</project_license>
   <name>Flameshot</name>
   <releases> 
+    <release version="0.10.2" date="2021-11-13"/>
     <release version="0.10.1" date="2021-07-25"/>
     <release version="0.10.0" date="2021-05-16"/>
     <release version="0.9.0" date="2021-02-27"/>
```

After rebuilding with this patch the issue will be resolved.

**Comment by mmahmoudian** [comment]
@xvitaly thanks for the patch. But I think this is an unnecessary move at this point as the v0.10.2 was just a hotfix and releasing a new version just to fix the version info sounds like an overkill. Perhaps with your contribution in other issues we can soon release v0.11.0 and get back to normal situation.

**Comment by AbdelrahmanMohamedDesoky** [comment]
Is reporting false version info and trolling your users with auto update notification to update to 0.10.2 fun for almost a month?

This issue is not even reported/acknowledged yet on the release page (where you could at least tell users to ignore the auto update notifications) ?

I know you are all busy and have work and stuff, but these type of issues is what drive people crazy sometimes, especially people who don't have the experience to find the GitHub repo of the app and search the issues listing to find the stupid bug report and find it.

**Comment by mmahmoudian** [comment]
@AbdelrahmanMohamedDesoky thanks for your kind and considerate comment. The version 0.10.2 was just to fix some issues Gnome caused by changing their API. So this is NOT "false version" and is clearly NOT "trolling". Also this will take as much as it takes for us to get prepared for v0.11.0. Therefore, if you are annoyed you can do either of the following:
1. Update to v0.10.2
2. Press ignore on the notification
3. Set the `ignoreUpdateToVersion=0.10.2` in your Flameshot config file

None of the above would change your experience with Flameshot other than preventing the update notice messages from appearing.

This is acknowledged in multiple places, but the release page is clearly not a place to mention such things.

Also I believe it is once again essential to point out the obvious:
we have pinned this issue (check the message above your comment) or two reasons:
1. Be easy to find and the first thing someone sees
2. To acknowledge that this is an important issue

Please feel free to keep suggesting enhancements and bugs [constructively] so that we can ultimately have a better Free and Libre OpenSource screenshot software :)

**Comment by razzeee** [comment]
The flatpak on flathub also says its 0.10.1

**Comment by mmahmoudian** [comment]
This was addressed in the #2161 PR, but since the release of v0.11.0 is imminent, it was decided that we just move forward with the release rather than a point-fix. That said, big thank you to @johnrichardrinehart for following up the case and create the PR.

I'll close this issue as the new version with fix it.

**Comment by johnrichardrinehart** [comment]
Good job you guys!
